.github/workflows/test-and-deploy.yml

```diff
@@ -53,7 +55,7 @@ jobs:
   deploy:
     if: github.repository_owner == 'compiler-explorer' && github.event_name == 'push'
     needs: [test, build_dist]
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
because this is still using npm to run some sentry commands, it might be better for this to remain 20.04 for now